packages/docs/docs/tutorials/data-modeling/family-relationships.md

```diff
@@ -0,0 +1,278 @@
+| ----------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [**Patient**](/docs/api/fhir/resources/patient)             | This is the resource that models a human who receives clinical services, such as diagnostic labs, medications, and procedures.                                                                                                                                                                                                                                                                                                                                                                                                     |
+| [**RelatedPerson**](/docs/api/fhir/resources/relatedperson) | This models the _relationship between two people_. For simple use cases, you can store demographic and contact information on the [RelatedPerson](/docs/api/fhir/resources/relatedperson) resource itself (see below) For more complex family models, you might want to store that information in a[Person](/docs/api/fhir/resources/person) or[Person](/docs/api/fhir/resources/patient)resource, and use the [RelatedPerson](/docs/api/fhir/resources/relatedperson) purely to model the relationship between the two resources. |
+| [**Person**](/docs/api/fhir/resources/person)               |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| [**Group**](/docs/api/fhir/resources/group)                 |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+
+## Decision Guide
```

Comment:
I'd add another table to make this more comprehensible.  Something like "common in" - newborn/maternal care, adolescent, behavioral health, pediatric specialty care.  Similar to https://www.medplum.com/products/integration#examples-integrations-by-interface-type